src/common/snippets/src/pass/collapse_subgraph.cpp

```diff
@@ -249,14 +218,10 @@ bool TokenizeSnippets::AppropriateForSubgraph(const std::shared_ptr<const Node>
 
 TokenizeSnippets::TokenizeSnippets(const SnippetsTokenization::Config& config) {
     MATCHER_SCOPE(TokenizeSnippets);
-    enum continuation_strategy {
```

Comment:
Note: tokenization code is moved without changes to tokenization_utils.cpp file with one exception: enum `continuation_strategy` is removed since it was used only with one value: `continuation_strategy::reset`